sdks/python/apache_beam/pipeline.py

```diff
@@ -299,6 +308,42 @@ def apply(self, transform, pvalueish=None, label=None):
     self.transforms_stack.pop()
     return pvalueish_result
 
+  def _verify_runner_api_compatible(self):
+    class Visitor(PipelineVisitor):  # pylint: disable=used-before-assignment
+      ok = True  # Really a nonlocal.
```

Comment:
It's an attribute (adding a getter method would probably be overkill). The comment is just that I'd use a non-local if it were available in Python 2.